### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by dimarogkov**
[DEMO LINK](https://dimarogkov.github.io/react_list-of-goods-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,45 +17,110 @@ export const goodsFromServer = [
...
+  const [reverseValue, setReverseValue] = useState(false);
+  const [resetValue, setResetValue] = useState(false);
+
+  const SORT_BY_ALPHABETICALLY = 'alphabetically';
+  const SORT_BY_LENGTH = 'length';
+
+  const sortBy = (value) => {
+    if (value === SORT_BY_ALPHABETICALLY) {
+      if (reverseValue) {
+        setGoodsArr([...goodsArr].sort((a, b) => b.localeCompare(a)));
```
Comment (mykhalenych) [inline src/App.jsx:32]: you can use previous value of state, and better will be create sort with if else, instead of if/else and two sorts

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 2]
```diff
@@ -14,45 +17,110 @@ export const goodsFromServer = [
...
+    setGoodsArr([...goodsArr].reverse());
+    setReverseValue(!reverseValue);
+  };
+
+  return (
+    <div className="section content">
+      <div className="buttons">
+        <button
+          type="button"
+          className={cn('button', 'is-info', {
```
Comment (etojeDenys) [inline src/App.jsx:78]:
```suggestion
          className={cn('button is-info', {
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]
fix prev comment then re-request me

File: src/App.jsx (hunk 1 above)
Comment (etojeDenys) [reply src/App.jsx:32]: still not fixed

Comment (etojeDenys) [reply src/App.jsx:32]: image

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Comment by dimarogkov** [comment]
Я не розумію, що від мене хочуть, поясніть ще раз

File: src/App.jsx (hunk 1 above)
Comment (VitaliyBodnar) [reply src/App.jsx:32]: please use this arctice as example https://legacy.reactjs.org/docs/hooks-reference.html#usestate

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

**Review by etojeDenys — CHANGES_REQUESTED** [review]
look in the slack thread

**Pull request closed without merging**